test/test_autograd.py

```diff
@@ -7765,6 +7764,24 @@ def test_out_variant(self):
             with self.assertRaisesRegex(RuntimeError, "out= function"):
                 torch.add(foo, bar, out=bar)
 
+    # TODO: Temporary solution to substitute for OpsInfo
+    def test_mkldnn_convolution(self):
+        inp = torch.rand(2, 2, 5, 5, dtype=torch.float, requires_grad=True)
+        weight = torch.rand(2, 2, 3, 3, dtype=torch.float, requires_grad=True)
+        bias = torch.rand(2, dtype=torch.float, requires_grad=True)
+
+        padding = (2, 3)
+        stride = (1, 1)
+        dilation = (3, 1)
+        groups = 1
+
+        torch.autograd.gradcheck(
+            torch.mkldnn_convolution,
+            [inp, weight, bias, padding, stride, dilation, groups],
+            eps=1e-3,
```

Comment:
Added 1e-3 because of this post - https://discuss.pytorch.org/t/why-is-nn-linear-failing-gradcheck-is-there-a-bug-in-gradcheck/17346/2

gradcheck uses double precision while `mkldnn_convolution` requires float or b16

Reply:
Yes, that's unfortunate...